tests/js-modules/modules.py

```diff
@@ -766,6 +773,66 @@ def test_npm_app(network, args):
     return network
 
 
+@reqs.description("Test JS execution time out with npm app endpoint")
+def test_js_execution_time(network, args):
+    primary, _ = network.find_nodes()
+
+    LOG.info("Deploying npm app")
+    app_dir = os.path.join(PARENT_DIR, "npm-app")
+    bundle_path = os.path.join(
+        app_dir, "dist", "bundle.json"
+    )  # Produced by build step of test npm-app in the previous test_npm_app
+    network.consortium.set_js_app_from_json(primary, bundle_path)
+
+    LOG.info("Store JWT signing keys")
+    jwt_key_priv_pem, _ = infra.crypto.generate_rsa_keypair(2048)
+    jwt_cert_pem = infra.crypto.generate_cert(jwt_key_priv_pem)
+    jwt_kid = "my_key_id"
+    issuer = "https://example.issuer"
+    with tempfile.NamedTemporaryFile(prefix="ccf", mode="w+") as metadata_fp:
+        jwt_cert_der = infra.crypto.cert_pem_to_der(jwt_cert_pem)
+        der_b64 = base64.b64encode(jwt_cert_der).decode("ascii")
+        data = {
+            "issuer": issuer,
+            "jwks": {"keys": [{"kty": "RSA", "kid": jwt_kid, "x5c": [der_b64]}]},
+        }
+        json.dump(data, metadata_fp)
+        metadata_fp.flush()
+        network.consortium.set_jwt_issuer(primary, metadata_fp.name)
+
+    LOG.info("Calling jwt endpoint after storing keys")
+    with primary.client("user0") as c:
+        user_id = "user0"
+        jwt = infra.crypto.create_jwt({"sub": user_id}, jwt_key_priv_pem, jwt_kid)
+
+        # set JS execution time to a lower value which will timeout this
+        # endpoint execution
+        network.consortium.set_js_runtime_options(
+            primary,
+            max_heap_bytes=50 * 1024 * 1024,
+            max_stack_bytes=1024 * 512,
+            max_execution_time=0.5,
```

Comment:
```suggestion
            max_execution_time=1,
```

Setting this to `0.5ms` should be disallowed by the action validation.